lib/modules/manager/gradle/extract/consistent-versions-plugin.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { stripIndent } from 'common-tags';
+import {
+  parseLockFile,
+  parsePropsFile,
+  usesGcv,
+} from './consistent-versions-plugin';
+
+describe('modules/manager/gradle/extract/consistent-versions-plugin', () => {
```

Comment:
* `parsePropsFile(`a.b:c.d=1\r\na.b:c.e=2`)[0].get('a.b:c.e');` will lead to null pointer exceptions in case `parsePropsFile` changes and eventually won't return [0]
* The tests are expected to return a concrete output. It should never "optionally" be null, e.g. `crlfProps2ndLine?`

Therefore, where it makes sense, please do test with `expect(...).toMatchObject` or a related approach.

Reply:
I hope I managed to rewrite the tests in a good way, e.g. asserting list of size 2 before working on `[0]` value.